src/server/services/generation/generation.service.ts

```diff
@@ -97,8 +97,42 @@ export const getGenerationResources = async ({
   user,
   supported,
 }: GetGenerationResourcesInput & { user?: SessionUser }): Promise<Generation.Resource[]> => {
+  const preselectedVersions: number[] = [];
+  if (!ids && !query) {
+    const featuredCollection = await dbRead.collection
+      .findFirst({
+        where: { userId: -1, name: 'Generator' },
+        select: {
+          items: {
+            select: {
+              model: {
+                select: {
+                  name: true,
+                  type: true,
+                  modelVersions: {
+                    select: { id: true, name: true },
+                    where: { status: 'Published' },
+                    orderBy: { index: 'asc' },
+                    take: 1,
+                  },
+                },
+              },
+            },
+          },
+        },
+      })
+      .catch(() => null);
```

Comment:
Doing a fallback to null here in case this query fails for some reason so it doesn't break the experience. It'll just fetch from all models in case this is ever null